ff/serving/metrics/test/metrics_test.go

```diff
@@ -75,22 +75,41 @@ func TestMetrics(t *testing.T) {
 		fmt.Println("error", err)
 	}
 	assert.Equal(t, int(servingCounterValue), servingNum, "5 feature rows should be served")
-
+	servingCounterValueInt, err := servingObserver.GetObservedRowCount()
+	if err != nil {
+		fmt.Println("error", err)
```

Comment:
If you change this to t.log() it'll show up above the test results in the end summary which will help with clarity